aws/logs_monitoring/tools/integration_tests/snapshots/cloudwatch_log_guardduty.json~snapshot

```diff
@@ -0,0 +1,3 @@
+{
+  "events": []
```

Comment:
Did you run `./tools/integration_tests/integration_tests.sh --update`? The snapshot shouldn't be empty.